.travis.yml

```diff
@@ -29,3 +29,11 @@ notifications:
   email:
     recipients:
       - sergio_89@ukr.net
+
+deploy:
+  provider: pypi
+  user: SergeyPirogov
+  password:
+    secure: encrypted-password
```

Comment:
ok, where to get this password?

Reply:
This is the password you use to log into http://pypi.org.

Reply:
I think we have to deploy on tag, it will give more control! I do same for other project